File: src/main/java/com/history/tree/dto/PersonDTO.java
package com.history.tree.dto;

import lombok.Data;

import java.time.LocalDate;

@Data
public class PersonDTO {

    private Long id;

    private String firstName;

    private String lastName;

    private String patronymic;

    private Long treeId;

    private Character gender;

    private LocalDate birthDate;

    private LocalDate dateOfDeath;
}

File: src/main/kotlin/com/history/tree/dto/PersonDto.kt
package com.history.tree.dto

import java.time.LocalDate
import java.util.*

data class PersonDto(
        val id: UUID,
        val treeId: UUID,
        val firstName: String,
        val lastName: String,
        val patronymic: String?,
        val gender: Char,
        val birthDate: LocalDate,
        val dateOfDeath: LocalDate?
)